src/libraries/System.Runtime.Extensions/tests/System/IO/PathTests.cs

```diff
@@ -234,6 +232,12 @@ public void GetFullPath_InvalidArgs()
                     { root + new string(Path.DirectorySeparatorChar, 3), root },
                 };
 
+                if (currentDirectory != Path.GetPathRoot(currentDirectory))
```

Comment:
Why do we need to do it this way, instead of just checking if we're not running in Browser?

Reply:
it's cleaner as this isn't really about browser but rather what currentDirectory you're running the test in. E.g. if you ran this test in `C:\` on Windows it wouldn't work either.

Reply:
I thought this was specific of Browser, since we didn't need this before.